azurerm/internal/services/web/app_service.go

```diff
@@ -1637,6 +1648,10 @@ func expandAppServiceSiteConfig(input interface{}) (*web.SiteConfig, error) {
 		siteConfig.HealthCheckPath = utils.String(v.(string))
 	}
 
+	if v, ok := config["number_of_workers"]; ok {
+		siteConfig.NumberOfWorkers = utils.Int32(int32(v.(int)))
+	}
```

Comment:
If the schema above is `Computed` without a default, we'll need to prevent the default for TypeInt (`0`) being sent here. e.g.
```suggestion
	if v, ok := config["number_of_workers"]; ok && v.(int) != 0 {
		siteConfig.NumberOfWorkers = utils.Int32(int32(v.(int)))
	}
```

Reply:
Looks good :-D